### ISSUE TITLE
Error in starting up the server

### ISSUE THREAD

**Issue by autholykos** [Sunny-Pirate/pastry-showcase-counter#1]
Following the instructions on the README.md I encountered a roadblock:

```
❯ npm start

> pastry-showcase-counter-nextjs@0.2.2 start
> next start

sh: next: command not found
```

It seems like installation of `next` should be listed as a requirement.

**Comment by autholykos** [comment]
Installing `next` locally (since globally bumps into another issue) seems to fix the above issue. The server seems to start, however it bumps into another issue:

```
❯ npm install next

added 331 packages, and audited 332 packages in 34s

104 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
❯ npm start

> pastry-showcase-counter-nextjs@0.2.2 start
> next start

ready - started server on 0.0.0.0:3000, url: http://localhost:3000
Error: Could not find a production build in the '/Users/emanuele/devel/other/pastry-showcase-counter/.next' directory. Try building your app with 'next build' before starting the production server. https://nextjs.org/docs/messages/production-start-no-build-id
    at NextNodeServer.getBuildId (~/devel/other/pastry-showcase-counter/node_modules/next/dist/server/next-server.js:172:23)
    at new Server (~/devel/other/pastry-showcase-counter/node_modules/next/dist/server/base-server.js:58:29)
    at new NextNodeServer (~/devel/other/pastry-showcase-counter/node_modules/next/dist/server/next-server.js:67:9)
    at NextServer.createServer (~/devel/other/pastry-showcase-counter/node_modules/next/dist/server/next.js:143:16)
    at async ~/devel/other/pastry-showcase-counter/node_modules/next/dist/server/next.js:155:31
    at async NextServer.prepare (~/devel/other/pastry-showcase-counter/node_modules/next/dist/server/next.js:130:24)
    at async ~/devel/other/pastry-showcase-counter/node_modules/next/dist/cli/next-start.js:118:9
```

**Comment by autholykos** [comment]
The last problem was solved by running the build. The server seems to start correctly now. The README should be updated to list the additional installation steps.

**Comment by Sunny-Pirate** [comment]
The problem is in the README commands file. I forgot to change README accordingly to the repo.

SOLUTION:
When cloned the repo install the dependencies with `npm install` and then run the application:
In development: `npm run dev`
In production: `npm run build && npm run start`.

I updated the README.
Thanks
